### PR TITLE
add enabledWorlds config option

### PR DESCRIPTION
**Pull request by WakkaFlocka239**

### CODE REVIEW

**Comment by bergerkiller** [comment]
Wait, but `disabledWorlds` already exists. Won't this be super confusing? What if the list is empty, is it then enabled on all worlds? Or disabled on all worlds?

File: src/main/java/com/bergerkiller/bukkit/tc/TCConfig.java [hunk 1]
```diff
@@ -341,7 +342,18 @@ public static void load(FileConfiguration config) {
         config.addHeader("activatorEjectEnabled", "If activator rails are used for decoration purposes, this should be disabled");
         activatorEjectEnabled = config.get("activatorEjectEnabled", true);
 
+        config.setHeader("enabledWorlds", "\nA list of world names where TrainCarts should be enabled");
+        config.addHeader("enabledWorlds", "World names are not case-sensitive");
+        enabledWorlds.clear();
+        if (!config.contains("enabledWorlds")) {
+            config.set("enabledWorlds", Arrays.asList("DefaultWorld1", "DefaultWorld2"));
```
Comment (bergerkiller) [inline src/main/java/com/bergerkiller/bukkit/tc/TCConfig.java:349]: Dont set to DefaultWorld1/DefaultWorld2, set to Collections.emptyList() instead.

File: src/main/java/com/bergerkiller/bukkit/tc/TCConfig.java [hunk 2]
```diff
@@ -341,7 +342,18 @@ public static void load(FileConfiguration config) {
         config.addHeader("activatorEjectEnabled", "If activator rails are used for decoration purposes, this should be disabled");
         activatorEjectEnabled = config.get("activatorEjectEnabled", true);
 
+        config.setHeader("enabledWorlds", "\nA list of world names where TrainCarts should be enabled");
+        config.addHeader("enabledWorlds", "World names are not case-sensitive");
```
Comment (bergerkiller) [inline src/main/java/com/bergerkiller/bukkit/tc/TCConfig.java:346]: Needs a comment explaining that an empty list means TrainCarts is enabled on all worlds of the server.

**Comment by WakkaFlocka239** [comment]
> Wait, but `disabledWorlds` already exists. Won't this be super confusing? What if the list is empty, is it then enabled on all worlds? Or disabled on all worlds?

I added this because I would much rather do


instead of this



I've addressed your comments, lmk if there's anything I missed

**Comment by bbayu123** [comment]
I honestly think using a flag to change the list would be more appropriate, for example, a flag to change the blacklist into a whitelist. Up to @bergerkiller to decide though.

**Pull request merged**